shared/downloads/jeos-23-64.ini

```diff
@@ -1,6 +1,6 @@
 [jeos-23-64]
 title = JeOS 23 x86_64
-url = http://assets-avocadoproject.rhcloud.com/static/jeos-23-64.qcow2.7z
+url = http://assets-avocadoproject.rhcloud.com/static/jeos-23-64.qcow2.xz
 sha1_url = http://assets-avocadoproject.rhcloud.com/static/SHA1SUM_JEOS23
```

Comment:
This should become, for a brief transition period:

https://assets-avocadoproject.rhcloud.com/static/SHA1SUM_JEOS23_XZ

The idea is to support users with and without this change for a few days (both the .7z and .xz images will be there too).